### PR TITLE
new version pre-update invisibly

### PR DESCRIPTION
**Pull request by lee1043**
Prepare for IM review

### CODE REVIEW

File: staff/index.md [hunk 1]
```diff
@@ -6,45 +6,39 @@ group: "navigation"
...
-[Nadeau, Denis][nadeau] | Computer Scientist | nadeau1@llnl.gov | (925) 423-3123
-[Painter, Jeff][painter] | Computer Scientist	| painter1@llnl.gov	| (925) 422-8114
-[Phillips, Tom][phillips] | Research Scientist | phillips14@llnl.gov | (925) 422-0072
-[Santer, Ben][santer] | Research Scientist | santer1@llnl.gov	|
-[Sperber, Ken][sperber] | Research Scientist | sperber1@llnl.gov | (925) 422-7720
-[Williams, Dean][williams] | Computer Scientist	| williams13@llnl.gov	| (925) 423-0145
-[Xie, Shaocheng][xie] | Research Scientist	| xie2@llnl.gov	| (925) 422-6023
+Name | Position 
+--- | --- 
+[Taylor, Karl][taylor] | Program Director
```
Comment (durack1) [inline staff/index.md:11]: Should be updated to "Staff Scientist" @taylor13 is no longer the director

File: staff/index.md [hunk 2]
```diff
@@ -6,45 +6,39 @@ group: "navigation"
...
+[Covey, Curt][covey] | Research Scientist 
+[Durack, Paul][durack] | Research Scientist	
+[Gleckler, Peter][gleckler] | Research Scientist 
+[Hoang, Anthony][hoang] | Systems Manager	
+[Jefferson, Angela][jefferson] | Program Administrator 
+[Lee, Jiwoo][lee] | Research Scientist 
+[Painter, Jeff][painter] | Computer Scientist	
+[Phillips, Tom][phillips] | Research Scientist 
+[Santer, Ben][santer] | Research Scientist 
+[Sperber, Ken][sperber] | Research Scientist
```
Comment (durack1) [inline staff/index.md:25]: Ken is no longer employed, so should be removed

File: staff/index.md [hunk 3]
```diff
@@ -6,45 +6,39 @@ group: "navigation"
...
+[Durack, Paul][durack] | Research Scientist	
+[Gleckler, Peter][gleckler] | Research Scientist 
+[Hoang, Anthony][hoang] | Systems Manager	
+[Jefferson, Angela][jefferson] | Program Administrator 
+[Lee, Jiwoo][lee] | Research Scientist 
+[Painter, Jeff][painter] | Computer Scientist	
+[Phillips, Tom][phillips] | Research Scientist 
+[Santer, Ben][santer] | Research Scientist 
+[Sperber, Ken][sperber] | Research Scientist 
+[Xie, Shaocheng][xie] | Research Scientist
```
Comment (durack1) [inline staff/index.md:26]: It seems you removed the directory that this points to, Shaocheng is not part of the PCMDI program

File: staff/index.md [hunk 4]
```diff
@@ -6,45 +6,39 @@ group: "navigation"
...
+[Bonfils, Celine][bonfils] | Research Scientist 
+[Caldwell, Peter][caldwell] | Research Scientist 
+[Covey, Curt][covey] | Research Scientist 
+[Durack, Paul][durack] | Research Scientist	
+[Gleckler, Peter][gleckler] | Research Scientist 
+[Hoang, Anthony][hoang] | Systems Manager	
+[Jefferson, Angela][jefferson] | Program Administrator 
+[Lee, Jiwoo][lee] | Research Scientist 
+[Painter, Jeff][painter] | Computer Scientist	
+[Phillips, Tom][phillips] | Research Scientist
```
Comment (durack1) [inline staff/index.md:23]: Both Curt and Tom are now retired, so should be noted as such

**Comment by lee1043** [comment]
@durack1 this PR didn't intended to update staff info. It was mostly for new PMP results. Let me close this to separate those updates.

File: staff/index.md [hunk 5]
```diff
@@ -6,45 +6,39 @@ group: "navigation"
...
+[Taylor, Karl][taylor] | Program Director 
+[Ahn, Min-Seop][ahn] | Post-doc 
+[Ames, Sasha][ames] | Computer Scientist 
+[Bonfils, Celine][bonfils] | Research Scientist 
+[Caldwell, Peter][caldwell] | Research Scientist 
+[Covey, Curt][covey] | Research Scientist 
+[Durack, Paul][durack] | Research Scientist	
+[Gleckler, Peter][gleckler] | Research Scientist 
+[Hoang, Anthony][hoang] | Systems Manager	
+[Jefferson, Angela][jefferson] | Program Administrator
```
Comment (durack1) [inline staff/index.md:20]: Angela's correct title is "Climate Program Office Manager"

File: staff/ahn/index.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Ahn
+---
+
+<p>
+<!--<a href="Lee-Ji-Woo-220x275-LLNL.jpg"><img src="Lee-Ji-Woo-220x275-LLNL.jpg" height="275" align="RIGHT" border="1"></a>-->
```
Comment (durack1) [inline staff/ahn/index.html:7]: You may want to reference a photo that looks like Min-Seop 😄

**Comment by lee1043** [comment]
@durack1 thanks for the checking! I will make another branch for this update.

**Pull request closed without merging**